### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by gawinwong** [wsvn53/scrcpy-mobile#2]
I can use the previous version to control my android from iPad. But I don't know how to work with this version. I have downloaded the "Scrcpy Remote" from App Store. How to set vnc server or something else. And How to use adb mode, I set VNC host "scrcpy2://adb" and port "5555", but it does work. Need further help

**Comment by wsvn53** [comment]
Please follow the instructions in README.md, if you want to connect with adb, please visit scrcpy2://adb in your Safari. Normally you will see a prompt, then restart the app to open it adb form.

**Comment by gawinwong** [comment]
Thanks for your reply, I have made a successful connection by installing an app named "droidVNC-NG" to my android phone. But the framerate and latency made me feel a little disappointed. Are there any settings to improve the bitrate and framerate, and if there are better vnc server apps to use to make the connection more smoothly?

**Comment by wsvn53** [comment]
Sorry, are you still in the vnc mode? Have you changed the mode to adb mode? It does not depends on any other apps on your Android.

Can you confirm the interface of Scrcpy Remote like this:

![IMG_20220621_104802_630.jpg](https://user-images.githubusercontent.com/912204/174705538-a9c3c3ce-bf35-401c-825b-68c59588fc89.jpg)

**Comment by gawinwong** [comment]
I have tried both two modes, and when it's in vnc mode I use an app to act as a vnc server. Under adb mode I just follow the previou guides.

**Comment by wsvn53** [comment]
> I have tried both two modes, and when it's in vnc mode I use an app to act as a vnc server. Under adb mode I just follow the previou guides.

So have you successfully connected your screen in adb mode? In addition, VNC mode is mainly used for arraignment, not the main function of this app, and the performance is also very poor.

**Comment by gawinwong** [comment]
Yes, both two modes are working well. And I will use adb mode to test game performence. Thanks for your efforts to spawn such a great app.
